tests/hooks/useRecentSearches.test.tsx

```diff
@@ -1,6 +1,5 @@
 import { useRecentSearches } from '../../src/hooks/useRecentSearches';
```

Comment:
(putting the comment here so we can have a comment chain, but asking about storyshot.test.ts.snap) What is storyshot.test.ts.snap?

Reply:
storyshot.test.ts.snap is a jest snapshot generated with stories using the storyshot add-on and would be used to compare against future renders of test UI in pr. (https://jestjs.io/docs/snapshot-testing) It needs to be updated by running `npm run coverage:visual` and committed if there is an approved change for the tests to pass.

Reply:
Is it possible to use the Percy snapshots instead of a local one? If not, no worries. But, I think we need to include a GH Action that will make a PR updating the Snapshot when appropriate. Without that action, people will inevitably forget to generate a new Snapshot and check it in.

Reply:
Do we need to add actual snapshot testing here? I thought all we wanted was the coverage, we don't need to see if the snapshot has changed.

If we do need this file and DO need it up to date I would rather generate it on the fly instead of checking it into the repo. I'd like to avoid adding additional noise to the repo if possible.

Reply:
Also is `storyshot.test.ts.snap` the default name for the file? It feels a little wordy